content/2.developers/4.cosmwasm-documentation/3.architecture/7.comparison-with-ethereum.md

```diff
@@ -18,12 +18,38 @@ Similarly to Ethereum:
 - Both instantiation and execution allow the signer to send some tokens to the contract alongside the message.
 
 Differently from Ethereum:
-- Sending tokens directly to a contract (i.e., via SendMsg) does not trigger any contract code. As contract execution needs to be explicitly requested. This CosmWasm design decision helps reduce possible attack vectors.
+- Sending tokens directly to a contract (i.e., via SendMsg) does not trigger any contract code. As contract execution needs to be explicitly requested. This CosmWasm design decision helps reduce possible attack vectors. For example inn Ethereum, when tokens are transferred to a contract, the contract's fallback function is activated automatically. If this fallback function is not well-implemented or has security vulnerabilities, it can leave the contract or even the entire system susceptible to different kinds of attacks, including reentrancy attacks.
```

Comment:
```suggestion
- Sending tokens directly to a contract (i.e., via SendMsg) does not trigger any contract code. As contract execution needs to be explicitly requested. This CosmWasm design decision helps reduce possible attack vectors. For example in Ethereum, when tokens are transferred to a contract, the contract's fallback function is activated automatically. If this fallback function is not well-implemented or has security vulnerabilities, it can leave the contract or even the entire system susceptible to different kinds of attacks, including reentrancy attacks.
```